### PR TITLE
Added Profile Readme details in data.json

### PR DESCRIPTION
**Pull request by smv1999**

### CODE REVIEW

**Review by Nitesh-thapliyal — COMMENTED** [review]
## Image URL must be of the ss uploaded in the Issue.

> To add your github profile
- create an issue, with the title as your name, and in the description put a screenshot of the customized GitHub profile page and then copy the link and that link you have to use in data.json .

File: data.json [hunk 1]
```diff
@@ -550,5 +550,11 @@
     "githubUsername": "Ibtihel-ouni",
     "imageUrl": "https://user-images.githubusercontent.com/58151963/107116883-97f86480-6876-11eb-8382-fc1c6ce06403.png",
     "issueNumber": 338
+  },
+  {
+    "name": "Vaidhyanathan S M",
+    "githubUsername": "smv1999",
+    "imageUrl": "https://drive.google.com/file/d/1YaiFeRqbPNzxNON3UjtPXaqiCmuNErXD/view?usp=sharing",
```
Comment (AllanRegush) [inline data.json:557]: Please paste your image into your issue message. It will give you a link to put in here 👍

**Review by AllanRegush — COMMENTED** [review]
Looking Good 👍 Just a minor adjustment 🤓

**Comment by smv1999** [comment]
@AllanRegush @Nitesh-thapliyal I have made the suggested changes.

**Review by Nitesh-thapliyal — APPROVED** [review]
Looks good👍Thank's for adding your profile🙌

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank you

**Comment by AllanRegush** [comment]
It looks like there is a conflict could you resolve this? If you need any help let me know. :nerd_face:

**Comment by smv1999** [comment]
I guess it is fine now @AllanRegush .

**Comment by AllanRegush** [comment]
It looks like someones entry got over ridden. If you need any help let me know :nerd_face:

**Review by AllanRegush — CHANGES_REQUESTED** [review]
It looks like  in the conflict someones entry got overridden.

**Comment by smv1999** [comment]
Really sorry about that @AllanRegush . I have created a new PR  #356  for the same as I faced some issues.

**Pull request closed without merging**